app_entry_script.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Wait for RabbitMQ to be available
+./wait-for-it.sh rabbitmq:$RABBITMQ_PORT --
+
+# Run the worker service
+exec /app/bin/worker-service
```

Comment:
This works, but if we will change the name of the working directory in docker file it will stop. There is no need to use absolute path. Consider using relative paths, since entry points script will always reside in the root of the application